### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by leonzfa** [ialhashim/DenseDepth#22]
Do you use a single learning rate (0.0001) at all experiments? If I train the network too much iterations, e.g., 100 epoch on NYUDepthv2 and then 100 epoch on KITTI, will the model forgets its pretraining on ImageNet?

**Comment by ialhashim** [comment]
It is a common problem in DNNs that training on different data leads the network to perform worse on earlier datasets.

I usually train for 20 epochs (after that you get little increase.. but you can try). What I do for learning rate is start with 0.0001 (or recently 0.00025) then after like 10 epochs divide it by 10. I only do this once and it seems to be a good schedule.
